ansible/roles/test/files/ptftests/py3/dhcp_relay_test.py

```diff
@@ -284,13 +356,14 @@ def create_dhcp_offer_relayed_packet(self):
                     ('server_id', self.server_ip),
                     ('lease_time', self.LEASE_TIME),
                     ('subnet_mask', self.client_subnet),
+                    ("vendor_class_id", "http://10.23.145.67:9876/abc/defghijk/lmnopqrst.bin".encode('utf-8')),
```

Comment:
>http://10.23.145.67:9876/abc/defghijk/lmnopqrst.bin [](http://example.com/codeflow?start=41&length=51)

Is it a random URL?
If yes, can we use more fictional URL like `[http://0.0.0.0/test/test.bin`](http://0.0.0.0/test/test.bin`) ?

Reply:
http://0.0.0.0/test/test.bin is not long enough to overwrite option82.

Reply:
This information is priceless, and please add it as code comment or assert, in case future dev (even yourself) mistakenly break it.

The the sample is like `[http://0.0.0.0/very_long_long_test/test.bin`](http://0.0.0.0/very_long_long_test/test.bin`)

Reply:
We do not want to a true IP in the world hit by DDOS.